### PR TITLE
Increases Resources for Space, and adds more ruins

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Increases the resources allocated for space, decreases the total zlevels and adds more ruins.

## Why It's Good For The Game

Space is fun!

## Changelog
:cl:
add: Increase the memory allocation for space ruin generation
add: Added new ruins
/:cl:

### CODE REVIEW

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -44,6 +43,78 @@
...
+	id = "sos7"
+	suffix = "sos7.dmm"
+	name = "Pod 7"
+	description = "Help! I was human once!"
+
+/datum/map_template/ruin/space/sos8
+	id = "sos8"
+	suffix = "sos8.dmm"
+	name = "Pod 8"
+	description = "CRASH!!!"
```
Comment (Archanial) [inline code/datums/ruins/space.dm:116]: Byond after increasing space resources be like :pensive:

File: code/modules/antagonists/eldritch_cult/eldritch_book.dm [hunk 2]
```diff
@@ -8,6 +8,8 @@
 	var/mob/living/last_user
 	///how many charges do we have?
 	var/charge = 1
+	///Where we cannot create the rune?
+	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava))
```
Comment (Archanial) [inline code/modules/antagonists/eldritch_cult/eldritch_book.dm:12]: I don't think it was meant to be pushed

**Comment by ghost** [comment]
I increased this once, about a year ago when Sage had a big influx if players, it was more recently reverted to it's normal state.

Ike or Crossed mentioned somewhere that they were concerned about the performance cost of more ruins.

https://github.com/BeeStation/BeeStation-Hornet/pull/1843

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Changelog changes, unintended changes to heretics and also 50???? Space ruins do not work like lavaland ruins, they all have a cost of 1. 50 is 50 space ruins.

**Comment by eeSPee** [comment]
> Changelog changes, unintended changes to heretics and also 50???? Space ruins do not work like lavaland ruins, they all have a cost of 1. 50 is 50 space ruins.

I did not know that.

Also unintended changes to heretics is because apparently I merged the wrong branch. Will look into that.

So, 16 resources = 16 ruins?

I could change the resource allocation (based on ruin size) or reduce the resources. Which one should it be?

**Comment by eeSPee** [comment]
Git shit itself

**Pull request closed without merging**